### DISCUSSION TITLE
preview screen feature

### DISCUSSION THREAD

**Discussion by og900aero** [gokcehan/lf#1859] [Q&A]
Someone managed to solve that in the preview image:
- wrap the contents of text files relative to the screen
- be able to scroll the contents of text files in the preview image, maybe the pages of pdfs?

**Comment by og900aero** [comment]
Wrap contents so-so solution:

in preview file use:

batcat --color=always --style=plain --pager=never --terminal-width $(($lf_width / 2 - 5)) --wrap character "$1"

Non-debian linux:
bat --color=always --style=plain --pager=never --terminal-width $(($lf_width / 2 - 5)) --wrap character "$1"

**Reply by CatsDeservePets** [reply]
Hey @og900aero!
I see you were also active in https://github.com/gokcehan/lf/discussions/1946. so can this be closed? 
Or is there still some stuff unanswered?

**Reply by og900aero** [reply]
> Hey @og900aero! I see you were also active in #1946. so can this be closed? Or is there still some stuff unanswered.

Lockable. Limited, but it works. It would be better if there was a built-in option for this, but you can still navigate with it.

**Reply by CatsDeservePets** [reply]
> Lockable. Limited, but it works. It would be better if there was a built-in option for this, but you can still navigate with it.

All right!

I will think about this a little more, perhaps there is a straight forward way of implementing this into `lf` by default.
